src/main/java/com/fryrank/controller/ReviewController.java

```diff
@@ -0,0 +1,26 @@
+package com.fryrank.controller;
+
+import com.fryrank.dal.ReviewDAL;
+import com.fryrank.model.Review;
+import lombok.NonNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ReviewController {
+
+    @Autowired
+    private ReviewDAL reviewDAL;
+
+    @GetMapping(value = "/reviews/")
```

Comment:
Not sure about standards, but do RESTful URL paths usually end in `/`?

Reply:
Yeah I think you're right. Will fix that